src/components/organisms/editorTabs.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useCallback, useEffect, useState } from "react";
 
 import Editor, { Monaco } from "@monaco-editor/react";
+import Cookies from "js-cookie";
```

Comment:
why is `cookie` but not `localStorage`?

Reply:
You are right, localStorage is better. Refactored.